jjb/mkck/mkck-trigger.yaml

```diff
@@ -139,15 +139,6 @@
             ref: 'ses7'
             repo: 'https://github.com/suse/ceph.git'
             ses_ver: '7.0'
-        - 'tumbleweed-ceph':
-            dist_name: 'tumbleweed'
-            dist_ver: 'xxxx'
-            disabled: false
```

Comment:
Could you just to set `disable: true`
So it does not run daily.

Reply:
Why? we don't want it and I currently don't see the bandwidth to maintain it. So I would prefer to remove it. It's there in `git log`...

Reply:
What is the reason of removing it totally, I periodically hear request to test something on tumbleweed.

Reply:
Not the `requests` but wishes.